src/components/Title/Title.scss

```
@@ -0,0 +1,17 @@
+.title{
+    @media screen and (max-width: $breakpoint-mobile) {
+      text-align: center;
+    }
+
+    &__left {
+      text-align: start;
+    }
+
+    &__center {
+      text-align: center;
+    }
+
+    &__right {
+      text-align: end;
+    }
+}
```

Needs a line at bottom of file

```suggestion
}

```